server/response.txt

```diff
@@ -0,0 +1,27 @@
+Hello, welcome to Canada Goose!
```

Comment:
quick question is this response.txt used in the code? i.e., do we pass this into the gpt response as an example?
if not i think we should remove it from the repo (but feel free to keep it on your local)